instrumentation/kafka/kafka-clients/kafka-clients-2.6/metadata.yaml

```diff
@@ -1,2 +1,13 @@
 description: >
-  This instrumentation provides a library integeration that enables messaging spans and metrics for Apache Kafka 2.6+ clients.
+  This instrumentation provides both library and wrapper integrations that enable messaging spans and metrics for Apache Kafka 2.6+ clients.
```

Comment:
By `library instrumentation` we mean a standalone instrumentation that can be used without the agent. `wrapper integration` isn't part of our terminology. I can understand that you mean that the library offers two kind of instrumentations, one that is based on interceptors and the other one that wraps producers and consumers, but for casual reader imo this sentence does not make sense.

Reply:
@laurit The word “wrapper” felt out of place since it wasn’t used anywhere in any metadata.yaml, so I agree it should be removed.